packages/react/src/hooks/useBlockNote.ts

```diff
@@ -41,7 +46,7 @@ export const useBlockNote = (
     //  but it would have to be on several different classes (BlockNoteEditor, BlockNoteEditorOptions, UiFactories) and
     //  gets messy quick.
     let newOptions: Record<any, any> = {
-      slashCommands: defaultReactSlashMenuItems,
+      slashCommands: defaultReactSlashMenuItems(),
```

Comment:
No point keeping it a function since it always returns the same thing. There are typing issues that I tried to solve by making it a function with a block schema type argument, but it was still bad. The issue I'm talking is this:

Example using custom alert block:
```
const editor = useBlockNote({
    blockSchema: {
      ...defaultBlockSchema,
      alert: Alert
    },
    slashCommands: [
      // Expects editor schema to be defaultBlockSchema. Structurally, it is, but also includes the 
      // alert block. TypeScript for some reason thinks that the schema is therefore incompatible
      // with the default slash commands.
      ...defaultReactSlashMenuItems,
      insertAlert
    ]
});
```

This will cause a type error in your IDE. Remove the alert block and its slash command, the error goes away. Not sure why this is happening since the custom schema extends the default schema.

Reply:
Tried to reproduce this in `custom-blocks-example-try-reproduce-type-issue` with a simpler example, but without luck.